### DISCUSSION TITLE
A general question about adobe ID, acsm, and the recommendations

### DISCUSSION THREAD

**Discussion by frere-jacques** [Leseratte10/acsm-calibre-plugin#75] [Q&A] [answered]
I am really new to eBooks and this particular kind of DRM. So maybe I don't understand the recommendations fully. Maybe you can enlighten me.

So I ordered a book at Thalia. I have that listed in there online store and I guess if I would have a reader of their brand I maybe would never need an adobe ID? I don't know. At least my first puzzle was that I have that book, but I don't know whether this book or the .acsm file is already connected to me.

I then created an adobe ID and used ADE under wine (terrible experience to get this running) to get the still encrypted epub.

So I wonder now, how does thalia now know that I connected this book to this ID? If I would get a Tolino device or other would I need to specify it? What identifier prohibits it to just get a again a new .amcs file and load the book again?

And is there really something like a dummy adobe ID? If I would create a dummy ID and use it to get a new .amcs file, wouldn't the book be forever connected to this ID and not to the main ID? So how would that help?

And if I get the book anonymously, wouldn't it be impossible to load it ever again from adobe?

I got it going now, but your solution appears nicer. But the warning is confusing and I think I don't get it or even don't get the adobe DRM at all.

**Comment by Leseratte10** [comment, marked as answer]
A book bought at Thalia (or any other bookstore) may be linked to your Thalia account, but the ACSM file itself that you can download from them is not linked to any account or identity (yet). 

Thalia generated this ACSM file once, at purchase. You can download it from Thalia multiple times. 

However, the first time you open this file either in ADE or in my plugin, ADE (or my plugin) will contact the ADE server operated by Thalia. That does two things - it permanently links this ACSM file to the Adobe identity you used for this request, and it sends you a copy of the book that's encrypted for this particular Adobe identity. 

That means that if you later download the ACSM again and try to open it with a different Adobe identity, you will get an error message. 

Your understanding about the dummy ID and/or the anonymous ID is almost correct, you're just missing one detail. Yes, if you use a dummy ID, you won't be able to read the book elsewhere later. Yes, if you use an anonymous authorization it will be impossible to ever load this again from Adobe except for with this particular Calibre installation. 

However, and this is the key, most people combine my plugin with additional plugins to remove Adobe's DRM upon import. When doing that, you don't have to worry about Adobe accounts since any book downloaded from an ACSM will have its DRM removed automatically and from thereon, that EPUB file will be readable anywhere. Though, it does mean that you are responsible for making backups yourself - if you reinstall your PC and get a new anonymous identity, you will still be able to read existing books that had their DRM removed (since, well, the DRM's gone) but you will never be able to download that book from Thalia again unless you re-purchase it. 

The goal of the dummy ID or the anonymous ID was to prevent your main Adobe account (which you may be using for other Adobe features as well) from getting banned if Adobe ever detects you're using my plugin and no official client. So far, that's never happened so the risk is probably fairly low. So if you want to take that risk, feel free to authorize the plugin with your "real" Adobe account.

I hope that answers your question. If something's still unclear just ask again.

**Reply by frere-jacques** [reply]
Yes that clarifies it. Since I didn't already have an adobe ID and don't use any adobe products, I didn't saw any other products or previous books at risk. So my new ID that will most likely only be used for buying books and probably this library thing, is therefore in principle a dummy ID.

But although I have backup strategies and the sole idea is to not worry about using the ID, it's still nice if a future download would work.

Interedting that the vendors run these servers themself.

Maybe if I have to set it up again, I will switch to your solution :). Thanks for the quick reply.
